### PR TITLE
Added Connect 4 game application

### PR DESCRIPTION
**Pull request by Kozova1**
The game requires 2 players to play (alternating turns).

### CODE REVIEW

File: wasp/apps/connect4.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
+"""Connect 4
+   ~~~~~~~~~~~~~
+    
+    Connect 4 is a 2 player game played on a 6x7 (height x width) grid.
+    The players can choose in which column to drop a disc with their color,
+    and when one of the players gets 4 adjacent discs in a row, column or diagonal, they win.
+    
+    Connect 4 is a solved game, meaning that in every case,
+    by making the right move, the first player can win."""
```
Comment (daniel-thompson) [inline wasp/apps/connect4.py:9]: Please no indentation here please.

Also please add a screenshot ('s' in the simulator) to the header comment (see any of the apps in the [Application Library](https://wasp-os.readthedocs.io/en/latest/apps.html) for examples of how to mark this up.

File: wasp/apps/connect4.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+import wasp
+import array
+import draw565
```
Comment (daniel-thompson) [inline wasp/apps/connect4.py:13]: This import is unused.

**Review by daniel-thompson — CHANGES_REQUESTED** [review]
Thanks for the PR. Just a couple of review comments from me.

In addition to adding a screenshot to the header you could also add this to the application guide (docs/apps.rst).

Can you take a look at the DCO failure from the automatic checks? This basically means the commits are misisng Signed-off-by: tags. Check ou the [Contributor's Guide](https://wasp-os.readthedocs.io/en/latest/contributing.html) for more details.

File: wasp/apps/connect4.py (hunk 1 above)
Comment (Kozova1) [reply wasp/apps/connect4.py]: done

File: wasp/apps/connect4.py (hunk 2 above)
Comment (Kozova1) [reply wasp/apps/connect4.py]: removed

**Comment by Kozova1** [comment]
Added all changes requested, but not sure what to put in `docs/apps.rst`

**Comment by Kozova1** [comment]
hopefully those commits should fix it now

**Comment by Kozova1** [comment]
Sorry, messed this one up too

**Pull request closed without merging**